vpr/src/place/place.cpp

```diff
@@ -1089,7 +1251,7 @@ static void placement_inner_loop(float t,
             num_swap_rejected++;
         }
 
-        if (placer_opts.place_algorithm == PATH_TIMING_DRIVEN_PLACE) {
+        if (placer_opts.place_algorithm.is_timing_driven()) {
             /* Do we want to re-timing analyze the circuit to get updated slack and criticality values?
```

Comment:
Both `place_algorithm` and `placer_opts.place_algorithm` are used?

Reply:
Sorry, it should be ```place_algorithm``` here instead of ```placer_opts.place_algorithm```